pytrip/utils/cubeslice.py

```diff
@@ -227,20 +231,31 @@ def main(args=sys.argv[1:]):
     ax.xaxis.set_minor_locator(minorLocator)
     ax.yaxis.set_minor_locator(minorLocator)
 
+    text_slice = ax.text(0.0, 1.05, "", size=8, transform=ax.transAxes)
```

Comment:
I might want to move it later if more info is added. In fact id like smth like the pytripgui rendering. But this really not important right now..